src/webchat-ui/components/presentational/chat-options/DeleteConversation.tsx

```diff
@@ -0,0 +1,122 @@
+import React from "react";
+import { useSelector, useDispatch } from "react-redux";
+import { Typography } from "@cognigy/chat-components";
+import Modal from "../../Modal/Modal";
+import { deletePrevConversation } from "../../../../webchat/store/previous-conversations/previous-conversations-reducer";
+import { getStorage } from "../../../../webchat/helper/storage";
+import { setShowChatOptionsScreen } from "../../../../webchat/store/ui/ui-reducer";
+import { StoreState } from "../../../../webchat/store/store";
+import styled from "@emotion/styled";
+import { IWebchatConfig } from "../../../../common/interfaces/webchat-config";
+import { clearMessages } from "../../../../webchat/store/messages/message-reducer";
+import Button from "../Button";
+import SecondaryButton from "../SecondaryButton";
+
+const Container = styled.div`
+	display: flex;
+	flex-direction: column;
+	align-items: flex-start;
+`;
+
+const DeleteButton = styled(Button)(({ theme }) => ({
+	background: theme.red,
+	color: theme.white,
+	width: "30%",
+	"&:hover:not(:disabled)": {
+		background: theme.red10,
+		color: theme.black10,
+	},
+}));
+
+const CancelButton = styled(SecondaryButton)(() => ({
+	marginRight: "auto",
+	width: "30%",
+}));
+
+const DeleteAnywaysButton = styled(Button)(({ theme }) => ({
+	color: theme.white,
+	background: theme.red,
+	marginLeft: "auto",
+	width: "30%",
+	"&:hover:not(:disabled)": {
+		background: theme.red10,
+		color: theme.black10,
+	},
```

Comment:
Hover styles of the buttons does not look consistent with the rest of the buttons. For example, start conversation button